tests/fixtures/optimizer/qualify_columns.sql

```diff
@@ -0,0 +1,127 @@
+--------------------------------------
+-- Qualify columns
+--------------------------------------
+SELECT a FROM x;
+SELECT "x"."a" AS "a" FROM x;
+
+SELECT a AS a FROM x;
+SELECT "x"."a" AS "a" FROM x;
+
+SELECT x.a FROM x;
+SELECT "x"."a" AS "a" FROM x;
+
+SELECT "x"."a" AS "a" FROM x;
+SELECT "x"."a" AS "a" FROM x;
+
+SELECT a AS b FROM x;
+SELECT "x"."a" AS "b" FROM x;
+
+SELECT 1, 2 FROM x;
+SELECT 1 AS "_col_0", 2 AS "_col_1" FROM x;
```

Comment:
sorry i forgot if i asked this but shouldn't this alsobe FROM x AS x

Reply:
Yeah, we agree we should do that.
But I think that should be done in `qualify_tables`, so perhaps in a separate PR.

Reply:
sounsd good